ga4gh/datamodel/variants.py

```diff
@@ -207,9 +208,13 @@ def getVariants(self, referenceName, startPosition, endPosition,
             callSetIds = self._sampleNames
         else:
             readCols = self._table.columns()[:self._firstSamplePosition]
-            for callSetId in callSetIds:
-                cols = [col for name, col in self._sampleCols[callSetId]]
-                readCols.extend(cols)
+            try:
+                for callSetId in callSetIds:
```

Comment:
I think the `try` should be as close to the source of the potential error as possible, e.g.
```python
for  callSetId in callSetIds:
    try:
          cols = [col for name, col in self._sampleCols[callSetId]]
    except KeyError:
          raise backendExceptions.CallSetNotInVariantSetException(...)
    readCols.extend(cols)
```